app/src/main/java/com/zeapo/pwdstore/git/GitOperation.kt

```diff
@@ -222,20 +220,13 @@ abstract class GitOperation(gitDir: File, internal val callingActivity: AppCompa
      */
     open fun onError(err: Exception) {
         // Clear various auth related fields on failure
-        when (SshSessionFactory.getInstance()) {
-            is SshApiSessionFactory -> {
-                PreferenceManager.getDefaultSharedPreferences(callingActivity.applicationContext)
-                    .edit { remove(PreferenceKeys.SSH_OPENKEYSTORE_KEYID) }
-            }
-            is SshjSessionFactory -> {
-                callingActivity.applicationContext
-                    .getEncryptedPrefs("git_operation")
-                    .edit {
-                        remove(PreferenceKeys.SSH_KEY_LOCAL_PASSPHRASE)
-                        remove(PreferenceKeys.HTTPS_PASSWORD)
-                    }
+        callingActivity.applicationContext
+            .getEncryptedPrefs("git_operation")
```

Comment:
we should probably also move this to an extension, call it `getGitCredentialPrefs` or something.